iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -314,6 +314,80 @@ TEST_F(PortManager_test, doDiscovery_rightOrdering)
     EXPECT_THAT(subscriber2.getSubscriptionState(), Eq(iox::SubscribeState::SUBSCRIBED));
 }
 
+TEST_F(PortManager_test, subscribeOnCreate_subscribesWithoutDiscoveryLoopWhenPublisherAvailable)
```

Comment:
Please use UpperCamelCase for the test names.

Reply:
I have adapted to my surroundings like a chameleon. So I'll go "all in" in this file